create_pnor_image.pl

```diff
@@ -99,6 +104,10 @@
 print "scratch_dir = $scratch_dir\n";
 print "pnor_data_dir = $pnor_data_dir\n";
 
+if($xz_compression eq "false") {
+    run_command("sed -i '/<xz/d'  $xml_layout_file\n");
```

Comment:
I'm not sure, but I don't think so.  We need this to be part of the ffs stuff, and this is how that's done.  I could make a special pnorLayout.xml file for when we're doing the xz stuff, and pull that file in instead of the defaultPnorLayout.xml, but then we would have to pass in the individual sections that are xz-compressed instead of just removing <xz/> from all sections if we're not allowing xz decompression.  
I could change a <xz>1</xz> flag to an <xz>0</xz> flag, but we're still editing the xml file.